plugins/coreboot/fu-coreboot-common.c

```diff
@@ -23,8 +23,20 @@ fu_plugin_coreboot_version_string_to_triplet (const gchar *coreboot_version,
 	guint cb_build = 0;
 	gint rc;
 
-	rc = sscanf (coreboot_version, "CBET%u %u.%u-%u", &cb_version, &cb_major,
-			&cb_minor, &cb_build);
+	if (g_str_has_prefix (coreboot_version, "CBET")) {
```

Comment:
Should this whole block be guarded by a custom flag and paired with a quirk perhaps?

Reply:
added a coreboot.quirk